packages/ts-client/src/webrtc/webRTCEndpoint.ts

```diff
@@ -326,14 +326,13 @@ export class WebRTCEndpoint<EndpointMetadata = any, TrackMetadata = any> extends
     this.remote.updateMLineIds(data.midToTrackId);
     this.local.updateMLineIds(data.midToTrackId);
 
-    Object.values(data.midToTrackId)
+    Object.keys(data.midToTrackId)
       .map((trackId) => {
         if (!trackId) throw new Error('TrackId is not defined');
-        if (typeof trackId !== 'string') throw new Error('TrackId is not a string');
 
         return trackId;
       })
-      .map((trackId) => this.local.getTrackByMidOrNull(trackId))
+      .map((mid) => this.local.getTrackByMidOrNull(mid))
```

Comment:
what does `mid` mean?

Reply:
In Session Description Protocol (SDP) it's `media identifier`, a unique identifier for media stream

Reply:
i feel `mediaId` would give me a better understanding what this variable contains, hbu?

Reply:
`mid` comes from RFC https://www.rfc-editor.org/rfc/rfc5888.html. I'm not sure if this change will introduce more confusion

Reply:
In that case - should it be updated in all `.map` functions here? above one is returning `trackId`